js/script.js

```diff
@@ -0,0 +1,185 @@
+let allTasks = [];
+let valueInput = '';
+let input = null;
+let content;
+let activeEditTask = null;
+
+const buttonTasksAdd = async () => {

```

Comment:
buttonTasksAdd - it may or may not be a button.
better "addTask"

Reply:
+